bin/.travis/prepare_unittest.sh

```diff
@@ -2,13 +2,6 @@
 
 # File for setting up system for unit/integration testing
 
-# Disable xdebug to speed things up as we don't currently generate coverge on travis
-# And make sure we use UTF-8 encoding
-if [ "$TRAVIS_PHP_VERSION" != "hhvm" ] ; then
-    phpenv config-rm xdebug.ini
-    echo "default_charset = UTF-8" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

Comment:
Note: these are needed for every job actually, not just unit tests. Dropped completely hhvm check because AFAICR it is no longer available on any Travis instance (or at least the one spawned for our `trusty` configs).